plugins/net_plugin/net_plugin.cpp

```diff
@@ -790,6 +790,14 @@ namespace eosio {
       bool is_blocks_only_connection()const { return connection_type == blocks_only; }
       bool is_transactions_connection() const { return connection_type != blocks_only; } // thread safe, atomic
       bool is_blocks_connection() const { return connection_type != transactions_only; } // thread safe, atomic
+      uint32_t get_peer_start_block_num() const { return peer_start_block_num.load(); }
+      uint32_t get_peer_head_block_num() const { return peer_head_block_num.load(); }
+      uint32_t get_last_received_blk_num() const { return last_received_blk_num.load(); }
```

Comment:
`block` vs `blk`. Should be consistent in variable and function names.

Reply:
Renamed.